### PR TITLE
New connection modal

### PR DESCRIPTION
**Pull request by KadenZipfel**
Closes https://github.com/protofire/omen-exchange/issues/1667

### CODE REVIEW

**Review by hexyls — APPROVED** [review]

**Comment by pimato** [comment]
`Waiting for Approval on Metamask` should be `Requesting permission on Metamask`

**Review by pimato — CHANGES_REQUESTED** [review]
connection list item should also have a bg color change on hover with the color #FCFDFD , for reference https://www.figma.com/file/aTivxuHmp2l6Sn8nho1Ngu/Playground?node-id=1821%3A174

**Comment by pimato** [comment]
lets change the bg when modal is visible from 6% black opacity to 50% black opacity:
`background-color: rgba(0, 0, 0, 0.50);`

**Review by pimato — APPROVED** [review]

File: app/src/components/modal/modal_connect_wallet/index.tsx
Comment (Mi-Lan) [inline app/src/components/modal/modal_connect_wallet/index.tsx:279]: Maybe we can give this spinner big property since it's  smaller then in designs...

**Review by Mi-Lan — CHANGES_REQUESTED** [review]
Works good! Very fast timing on this one :D Just a few minor cometics i found!

File: app/src/components/modal/modal_connect_wallet/index.tsx
Comment (KadenZipfel) [reply app/src/components/modal/modal_connect_wallet/index.tsx:56]: That variable has the wrong color

File: app/src/components/modal/modal_connect_wallet/index.tsx
Comment (KadenZipfel) [reply app/src/components/modal/modal_connect_wallet/index.tsx:40]: Yeah, the issue is that the svg contains a drop shadow, so I accounted for it by using a lower margin

**Review by Mi-Lan — APPROVED** [review]

**Pull request merged**
